__tests__/app.test.js

```diff
@@ -117,7 +116,77 @@ describe('PATCH reviews/id', () => {
             .then(({
                 body
             }) => {
-                expect(body.votes).toBe(2)
+                console.log(body)
+                expect(body.review.votes).toBe(2)
+            })
+    })
+    test('when passed a neg vote obj increase the current votes by amount stated', () => {
+
+        const vote = {
+            inc_votes: -1
+        }
+
+
+        return request(app)
+            .patch("/api/reviews/1")
+            .send(vote)
+            .expect(200)
+            .then(({
+                body
+            }) => {
+                console.log(body)
+                expect(body.review.votes).toBe(0)
+            })
+    })
+    test('when passed an invalid id return 404', () => {
+        const vote = {
+            inc_votes: 1
+        }
+
+        return request(app)
+            .get("/api/reviews/1000")
+            .send(vote)
+            .expect(404)
+            .then(({
+                res
+            }) => {
+                expect(res.text)
+                    .toBe("Sorry Review cant be found")
+            })
+    });
+    test('when passed incorrect data type for id', () => {
+        const vote = {
+            inc_votes: 1
+        }
+
+        return request(app)
+            .patch("/api/reviews/wrong")
+            .send(vote)
+            .expect(400)
+            .then(({
+                body
+            }) => {
+                console.log(body.msg)
+                expect(body.msg)
+                    .toBe("Sorry incorrect input")
+            })
+
+    })
+    test('when passed incorrect data type for inc_votes', () => {
+        const vote = {
```

Comment:
the test description says that it's testing for incorrect data type for inc_votes but you're passing in a correct inc_votes and passing the wrong review_id